.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+Replace this paragraph with a description of what this PR is doing. Include
+links to relevant issues. If you're modifying existing behavior, describe the
+existing behavior, how this PR is changing it, and what motivated the change. If
+you're changing visual properties, consider including before/after screenshots
+(and runnable code snippets to reproduce them).
+
+## Checklist
+
+Before you create this PR confirm that it meets all requirements listed below
+by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick
+review process.
+
+- [ ] I read the [Contributor Guide] and followed the process outlined there for
+      submitting PRs.
+- [ ] My PR includes tests for *all* changed/updated/fixed behaviors (See
+      [Test Coverage]).
+- [ ] All existing and new tests are passing.
```

Comment:
Reference the CI output since people may not know how breaking tests manifest

Reply:
The contributor guide above mentions how to do this. To keep this brief, I don't want to repeat this here...